mcp-auth/python/README.md

```diff
@@ -0,0 +1,160 @@
+# MCP Auth Python Quick Start Guide - Asgardeo Integration
+
+This project demonstrates how to create an authenticated MCP (Model Context Protocol) server using Python with Asgardeo as the OAuth2/OIDC provider and the FastMCP framework.
+
+## Features
+
+- JWT token validation using JWKS
+- Asgardeo OAuth2/OIDC integration
+- FastMCP framework with built-in authentication middleware
+- Custom JWTTokenVerifier implementation
+- Weather service with `get_weather` tool that requires authentication
+
+## Prerequisites
+
+- Python 3.12 or higher
+- Asgardeo account and application setup
```

Comment:
shall we add instruction on application setup (eg: setting PKCE, changing token type to JWT)?

Reply:
Let's link to `asgardeo configuration` section